docs/_sections/tables.html

```diff
@@ -384,19 +384,19 @@ <h3 id="pagination">Search Results Pagination</h3>
       <span class="nypl-example-text">Example</span>
 <div class="nypl-results-pagination">
```

Comment:
I think this should be a `<nav>` element, and it should have an `aria-label` of some kind